### ISSUE TITLE
I got error with an antibot

### ISSUE THREAD

**Issue by radek203** [AuthMe/AuthMeReloaded#1958]
I got error with an antibot
A lot of people were entering the server and I got this error
Standalone server with SQLITE

```
[18:00:56] [User Authenticator #876/ERROR]: Could not pass event AsyncPlayerPreLoginEvent to AuthMe v5.6.0-SNAPSHOT-b2337
org.bukkit.event.EventException: null
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[server.jar:git-Spigot-dcd1643-e60fc34]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[server.jar:git-Spigot-dcd1643-e60fc34]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:500) [server.jar:git-Spigot-dcd1643-e60fc34]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:482) [server.jar:git-Spigot-dcd1643-e60fc34]
	at net.minecraft.server.v1_12_R1.LoginListener$LoginHandler.fireEvents(LoginListener.java:270) [server.jar:git-Spigot-dcd1643-e60fc34]
	at net.minecraft.server.v1_12_R1.LoginListener$2.run(LoginListener.java:187) [server.jar:git-Spigot-dcd1643-e60fc34]
Caused by: java.lang.NullPointerException
	at fr.xephi.authme.service.AntiBotService.stopProtection(AntiBotService.java:115) ~[?:?]
	at fr.xephi.authme.service.AntiBotService.startProtection(AntiBotService.java:88) ~[?:?]
	at fr.xephi.authme.service.AntiBotService.shouldKick(AntiBotService.java:172) ~[?:?]
	at fr.xephi.authme.listener.OnJoinVerifier.checkAntibot(OnJoinVerifier.java:78) ~[?:?]
	at fr.xephi.authme.listener.PlayerListener.onAsyncPlayerPreLoginEventHighest(PlayerListener.java:146) ~[?:?]
	at sun.reflect.GeneratedMethodAccessor58.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_141]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_141]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:302) ~[server.jar:git-Spigot-dcd1643-e60fc34]
	... 5 more
```

**Comment by radek203** [comment]
@ljacqu Will it be fixed?

**Comment by ljacqu** [comment]
Yes, we'll need to fix it. Thanks for reporting this!

**Comment by sgdc3** [comment]
@radek203 can you please try to replicate the issue using the latest dev build?

**Comment by radek203** [comment]
@sgdc3 I don't know how it was created, I can only say that a lot of people wanted to enter the server and I manually turned off the protective antibot by command.

**Comment by ljacqu** [comment]
Yes, it's a bug on our side. From the code it's quite apparent that it's not thread safe, not to mention that we haven't touched the antibot feature since 2018...

**Comment by sgdc3** [comment]
@ljacqu I fixed the issue in #1992
